src_python/habitat_sim/simulator.py

```diff
@@ -431,18 +437,23 @@ def get_sensor_observations(
         else:
             return_single = False
 
-        for agent_id in agent_ids:
-            agent_sensorsuite = self.__sensors[agent_id]
-            for _sensor_uuid, sensor in agent_sensorsuite.items():
-                sensor.draw_observation()
-
-        # As backport. All Dicts are ordered in Python >= 3.7
+        # As backport. All Dicts are ordered in Python >= 3.7.
         observations: Dict[int, ObservationDict] = OrderedDict()
+
+        # Draw observations for classic non-batched renderer.
+        if not self.config.enable_batch_renderer:
```

Comment:
As a readability thing, it might be nice to see an else block here with a `pass` and a comment about the batch renderer being a work in progress or not implemented yet or whatever. This would help answer my question/confusion below ("how can we safely call...").